src/Storage/Synapse/Table.php

```diff
@@ -57,6 +54,45 @@ public function getFromStatement(): string
         return sprintf('SELECT %s FROM %s', $select, $this->getQuotedTableWithScheme());
     }
 
+    public function getFromStatementForStaging(bool $castValues): string
```

Comment:
prosim taky malou poznamku o co jde at to clovek nemusi spekulovat

Reply:
btw proc potrebujeme `castValues` ? imho tady https://github.com/keboola/php-db-import-export/pull/100/files#diff-30516a87c8a9aabd72a270561ae8015928fbf13787e7935038484a8dbf0bd5c7R98 by to mohlo byt 
```
if($importOptions->areSameTablesRequired()){$from =  $source->getFromStatementForStaging()} ne?

Reply:
To by ti neprošlo přes STAN protože SourceInterface tu metodu nemá, musíš testovat že je to Table. Ale trochu to okomentuju.